modules/react/checkbox/lib/Checkbox.tsx

```diff
@@ -360,6 +368,7 @@ export const Checkbox = createComponent('input')({
             disabled={disabled}
             variant={variant}
             paddingInlineStart={checkboxLabelDistance}
+            cursor="pointer"
```

Comment:
@jamesfan just noticed the label doesn't have the correct cursor even though it is clickable